recipes/robin-hood-hashing/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+
+from conans import ConanFile, tools
+
+class ConanRecipe(ConanFile):
+    name = "robin-hood-hashing"
+    description = "Faster and more efficient replacement for std::unordered_map / std::unordered_set"
+    topics = ("conan", "robin-hood-hashing", "header-only", "containers")
+    homepage = "https://github.com/martinus/robin-hood-hashing"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+
+    generators = "cmake"
+
+    _source_subfolder = "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
```

Comment:
It seems you are settings c++14 in test package, is this correct?
```suggestion
        os.rename(extracted_dir, self._source_subfolder)

    def configure(self):
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, 14)

    def package(self):
```

Reply:
It looks it supports down to 11.